README.md

```diff
@@ -2,7 +2,9 @@
 [![slack](https://img.shields.io/badge/slack-privado-5A34D9.svg?logo=slack)](https://join.slack.com/t/privado-community/shared_invite/zt-yk5zcxh3-gj8sS9w6SvL5lNYZLMbIpw)
 [![docs](https://img.shields.io/badge/docs-gitbook-brightgreen.svg?logo=gitbook)](https://docs.privado.ai)
 
-Privado is an open-source static code analysis tool to discover data flows in the code. It detects Personally Identifiable Information (PII) being processed and further maps the data flow from the point of collection to "sinks" such as external third parties, databases, logs, and internal APIs.
+Privado is an open-source static code analysis tool to discover data flows in the code. It detects more than 120 Personal Information data elements being processed and further maps the data flow from the point of collection to "sinks" such as external third parties, databases, logs, and internal APIs.
+
+The entire list of data elements detected by the tool can be found [here](https://github.com/Privado-Inc/privado/tree/main/rules/sources)
```

Comment:
Can we add the link in the previous para as follows?
```
It detects more than 120 Personal Information [data elements]
```

Please also update the link to point the newly added excel.